docs/usage/dataclasses.md

```diff
@@ -25,18 +25,26 @@ User(id=42, name='John Doe', signup_ts=datetime.datetime(2032, 6, 21, 12, 0))
 ```
 
 !!! note
-    Keep in mind that `pydantic.dataclasses.dataclass` is **not** a replacement for `pydantic.BaseModel` (with a small
-    difference in how [initialization hooks](#initialize-hooks) work). `pydantic.dataclasses.dataclass` provides a
-    similar functionality to `dataclasses.dataclass` with the addition of Pydantic validation. There are cases where
-    subclassing `pydantic.BaseModel` is the better choice.
+    Keep in mind that `pydantic.dataclasses.dataclass` is **not** a replacement for `pydantic.BaseModel`.
+    `pydantic.dataclasses.dataclass` provides a similar functionality to `dataclasses.dataclass` with the addition of
+    Pydantic validation.
+    There are cases where subclassing `pydantic.BaseModel` is the better choice.
 
     For more information and discussion see
     [pydantic/pydantic#710](https://github.com/pydantic/pydantic/issues/710).
 
-You can use all the standard _pydantic_ field types. Note, however, that arguments passed to constructor will be copied in order to perform validation and, where necessary coercion.
+Some differences between Pydantic dataclasses and `BaseModel` include:
 
-The schema can be accessed by [TypeAdapter](models.md#typeadapter).
-Also, fields that require a `default_factory` can be specified by either a `pydantic.Field` or a `dataclasses.field`.
+*  How [initialization hooks](#initialization-hooks) work
+*  [JSON dumping](#json-dumping)
+
+You can use all the standard Pydantic field types. Note, however, that arguments passed to constructor will be copied in
+order to perform validation and, where necessary coercion.
+
+To perform validation or generate a JSON schema on a Pydantic dataclass, you should now wrap the dataclass
+with a [`TypeAdapter`](models.md#typeadapter) and make use of its methods.
+
+Fields that require a `default_factory` can be specified by either a `pydantic.Field` or a `dataclasses.field`.
```

Comment:
Should these be bullet points? (Not a suggestion, a Q)

Reply:
Not in this context